src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java

```diff
@@ -79,24 +86,57 @@ private void createPanel() {
     layout = new CardLayout();
     panel = new JPanel(layout);
 
+    var notConnectedCardWrapper = new JPanel(new GridBagLayout());
```

Comment:
This `CurrentFileConnectedModePanel` class was probably badly named before and even more now. Not sure which name to pick instead